lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -80,6 +80,11 @@ sub _save_vars {
     close($fd);
 }
 
+sub indent_log_raw_output {
```

Comment:
I would prefer to use signatures in new code (which could make this a one-liner function).

Reply:
Done